### ISSUE TITLE
[Qualification]: PowerScale CSI v2.6.1 on OpenShift 4.14 

### ISSUE THREAD

**Issue by marbleman2** [dell/csm#1513]
### Operating System

OpenShift 4.14

### Container Orchestrator

PowerScale CSI v2.6.1

### Storage Platform

PowerScale 9.4.0.16

### Protocol

NFS

### Relevant log output

```shell
report-test-run-fe74a2e8:
Name: test-run-fe74a2e8
Host: [36mhttps://api.kcs-lab2.k8s.boeing.com:6443[0m
StorageClass: [93misilon[0m
Minimum and Maximum EntityOverTime charts:

[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/PodsCreatingOverTime.png[0m

[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/PodsReadyOverTime.png[0m

[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/PodsTerminatingOverTime.png[0m

[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/PvcsCreatingOverTime.png[0m

[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/PvcsBoundOverTime.png[0m

Tests:
--------------------------------------------------------------
1. TestCase: [36mMultiAttachSuite[0m
            Started:   2024-10-01 07:31:20.12456459 -0700 -0700
            Ended:     2024-10-01 07:33:58.154950504 -0700 -0700
            Result:    [32mSUCCESS[0m

            Stage metrics:
		    PVCAttachment:
			Avg: 17.371169877s
			Min: 17.371169877s
			Max: 17.371169877s
			Histogram:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/MultiAttachSuite1/PVCAttachment.png[0m
			BoxPlot:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/MultiAttachSuite1/PVCAttachment_boxplot.png[0m
		    PVCBind:
			Avg: 13.345526619s
			Min: 13.345526619s
			Max: 13.345526619s
			Histogram:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/MultiAttachSuite1/PVCBind.png[0m
			BoxPlot:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/MultiAttachSuite1/PVCBind_boxplot.png[0m
		    PVCCreation:
			Avg: 52.342845689s
			Min: 52.342845689s
			Max: 52.342845689s
			Histogram:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/MultiAttachSuite1/PVCCreation.png[0m
			BoxPlot:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/MultiAttachSuite1/PVCCreation_boxplot.png[0m
		    PVCDeletion:
			Avg: 4.933418149s
			Min: 4.933418149s
			Max: 4.933418149s
			Histogram:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/MultiAttachSuite1/PVCDeletion.png[0m
			BoxPlot:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/MultiAttachSuite1/PVCDeletion_boxplot.png[0m
		    PVCUnattachment:
			Avg: 1.59069589s
			Min: 1.59069589s
			Max: 1.59069589s
			Histogram:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/MultiAttachSuite1/PVCUnattachment.png[0m
			BoxPlot:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/MultiAttachSuite1/PVCUnattachment_boxplot.png[0m
		    PodCreation:
			Avg: 18.525569185s
			Min: 7.871634354s
			Max: 28.042340959s
			Histogram:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/MultiAttachSuite1/PodCreation.png[0m
			BoxPlot:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/MultiAttachSuite1/PodCreation_boxplot.png[0m
		    PodDeletion:
			Avg: 3.047936206s
			Min: 1.316705598s
			Max: 5.153938363s
			Histogram:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/MultiAttachSuite1/PodDeletion.png[0m
			BoxPlot:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/MultiAttachSuite1/PodDeletion_boxplot.png[0m
			EntityNumberOverTime:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/MultiAttachSuite1/EntityNumberOverTime.png[0m
--------------------------------------------------------------
2. TestCase: [36mScalingSuite[0m
            Started:   2024-10-01 07:31:20.124619507 -0700 -0700
            Ended:     2024-10-01 07:32:51.847858938 -0700 -0700
            Result:    [32mSUCCESS[0m

            Stage metrics:
		    PVCAttachment:
			Avg: 6.187659242s
			Min: 2.552703095s
			Max: 9.965887511s
			Histogram:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/ScalingSuite2/PVCAttachment.png[0m
			BoxPlot:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/ScalingSuite2/PVCAttachment_boxplot.png[0m
		    PVCBind:
			Avg: 8.598123886s
			Min: 6.870400456s
			Max: 10.957264298s
			Histogram:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/ScalingSuite2/PVCBind.png[0m
			BoxPlot:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/ScalingSuite2/PVCBind_boxplot.png[0m
		    PVCCreation:
			Avg: 21.882638261s
			Min: 18.812499061s
			Max: 26.251048099s
			Histogram:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/ScalingSuite2/PVCCreation.png[0m
			BoxPlot:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/ScalingSuite2/PVCCreation_boxplot.png[0m
		    PVCDeletion:
			Avg: 71.908519ms
			Min: 12.830151ms
			Max: 176.045664ms
			Histogram:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/ScalingSuite2/PVCDeletion.png[0m
			BoxPlot:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/ScalingSuite2/PVCDeletion_boxplot.png[0m
		    PVCUnattachment:
			Avg: 3.072134463s
			Min: 725.474176ms
			Max: 6.559209278s
			Histogram:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/ScalingSuite2/PVCUnattachment.png[0m
			BoxPlot:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/ScalingSuite2/PVCUnattachment_boxplot.png[0m
		    PodCreation:
			Avg: 31.737701307s
			Min: 31.215366941s
			Max: 32.260035674s
			Histogram:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/ScalingSuite2/PodCreation.png[0m
			BoxPlot:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/ScalingSuite2/PodCreation_boxplot.png[0m
		    PodDeletion:
			Avg: 6.360508307s
			Min: 5.33542554s
			Max: 7.385591074s
			Histogram:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/ScalingSuite2/PodDeletion.png[0m
			BoxPlot:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/ScalingSuite2/PodDeletion_boxplot.png[0m
			EntityNumberOverTime:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/ScalingSuite2/EntityNumberOverTime.png[0m
--------------------------------------------------------------
3. TestCase: [36mVolumeIoSuite[0m
            Started:   2024-10-01 07:31:20.124628945 -0700 -0700
            Ended:     2024-10-01 07:33:31.67892414 -0700 -0700
            Result:    [32mSUCCESS[0m

            Stage metrics:
		    PVCBind:
			Avg: 10.806402089s
			Min: 8.704954572s
			Max: 12.907849607s
			Histogram:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/VolumeIoSuite3/PVCBind.png[0m
			BoxPlot:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/VolumeIoSuite3/PVCBind_boxplot.png[0m
		    PVCCreation:
			Avg: 38.260501459s
			Min: 14.410966934s
			Max: 1m2.110035984s
			Histogram:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/VolumeIoSuite3/PVCCreation.png[0m
			BoxPlot:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/VolumeIoSuite3/PVCCreation_boxplot.png[0m
		    PVCDeletion:
			Avg: 11.947107ms
			Min: 10.379245ms
			Max: 13.514969ms
			Histogram:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/VolumeIoSuite3/PVCDeletion.png[0m
			BoxPlot:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/VolumeIoSuite3/PVCDeletion_boxplot.png[0m
		    PVCUnattachment:
			Avg: 26.251440969s
			Min: 756.251275ms
			Max: 51.746630664s
			Histogram:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/VolumeIoSuite3/PVCUnattachment.png[0m
			BoxPlot:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/VolumeIoSuite3/PVCUnattachment_boxplot.png[0m
		    PodCreation:
			Avg: 12.958663883s
			Min: 7.966140755s
			Max: 16.111067655s
			Histogram:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/VolumeIoSuite3/PodCreation.png[0m
			BoxPlot:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/VolumeIoSuite3/PodCreation_boxplot.png[0m
		    PodDeletion:
			Avg: 2.978407422s
			Min: 1.61954698s
			Max: 4.389352834s
			Histogram:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/VolumeIoSuite3/PodDeletion.png[0m
			BoxPlot:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/VolumeIoSuite3/PodDeletion_boxplot.png[0m
			EntityNumberOverTime:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/VolumeIoSuite3/EntityNumberOverTime.png[0m
--------------------------------------------------------------
4. TestCase: [36mCloneVolumeSuite[0m
            Started:   2024-10-01 07:31:20.124622597 -0700 -0700
            Ended:     2024-10-01 07:32:47.448675134 -0700 -0700
            Result:    [32mSUCCESS[0m

            Stage metrics:
		    PVCAttachment:
			Avg: 4.581254589s
			Min: 1.262478069s
			Max: 5.746482121s
			Histogram:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/CloneVolumeSuite4/PVCAttachment.png[0m
			BoxPlot:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/CloneVolumeSuite4/PVCAttachment_boxplot.png[0m
		    PVCBind:
			Avg: 13.694507141s
			Min: 9.965621046s
			Max: 16.734515081s
			Histogram:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/CloneVolumeSuite4/PVCBind.png[0m
			BoxPlot:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/CloneVolumeSuite4/PVCBind_boxplot.png[0m
		    PVCCreation:
			Avg: 20.816310681s
			Min: 11.427569497s
			Max: 25.152328098s
			Histogram:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/CloneVolumeSuite4/PVCCreation.png[0m
			BoxPlot:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/CloneVolumeSuite4/PVCCreation_boxplot.png[0m
		    PVCDeletion:
			Avg: 2.729371038s
			Min: 2.525601098s
			Max: 2.880656598s
			Histogram:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/CloneVolumeSuite4/PVCDeletion.png[0m
			BoxPlot:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/CloneVolumeSuite4/PVCDeletion_boxplot.png[0m
		    PVCUnattachment:
			Avg: 2.03990344s
			Min: 1.362520624s
			Max: 3.318708827s
			Histogram:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/CloneVolumeSuite4/PVCUnattachment.png[0m
			BoxPlot:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/CloneVolumeSuite4/PVCUnattachment_boxplot.png[0m
		    PodCreation:
			Avg: 28.05080555s
			Min: 19.911002657s
			Max: 31.77251956s
			Histogram:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/CloneVolumeSuite4/PodCreation.png[0m
			BoxPlot:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/CloneVolumeSuite4/PodCreation_boxplot.png[0m
		    PodDeletion:
			Avg: 1.834826945s
			Min: 1.624635523s
			Max: 2.009471303s
			Histogram:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/CloneVolumeSuite4/PodDeletion.png[0m
			BoxPlot:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/CloneVolumeSuite4/PodDeletion_boxplot.png[0m
			EntityNumberOverTime:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/CloneVolumeSuite4/EntityNumberOverTime.png[0m
--------------------------------------------------------------
5. TestCase: [36mVolumeExpansionSuite[0m
            Started:   2024-10-01 07:31:20.124637049 -0700 -0700
            Ended:     2024-10-01 07:32:17.824904592 -0700 -0700
            Result:    [32mSUCCESS[0m

            Stage metrics:
		    PVCAttachment:
			Avg: 3.24466788s
			Min: 3.24466788s
			Max: 3.24466788s
			Histogram:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/VolumeExpansionSuite5/PVCAttachment.png[0m
			BoxPlot:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/VolumeExpansionSuite5/PVCAttachment_boxplot.png[0m
		    PVCBind:
			Avg: 11.837583108s
			Min: 11.837583108s
			Max: 11.837583108s
			Histogram:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/VolumeExpansionSuite5/PVCBind.png[0m
			BoxPlot:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/VolumeExpansionSuite5/PVCBind_boxplot.png[0m
		    PVCCreation:
			Avg: 22.263778337s
			Min: 22.263778337s
			Max: 22.263778337s
			Histogram:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/VolumeExpansionSuite5/PVCCreation.png[0m
			BoxPlot:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/VolumeExpansionSuite5/PVCCreation_boxplot.png[0m
		    PVCDeletion:
			Avg: 4.899801248s
			Min: 4.899801248s
			Max: 4.899801248s
			Histogram:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/VolumeExpansionSuite5/PVCDeletion.png[0m
			BoxPlot:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/VolumeExpansionSuite5/PVCDeletion_boxplot.png[0m
		    PVCUnattachment:
			Avg: 3.734682767s
			Min: 3.734682767s
			Max: 3.734682767s
			Histogram:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/VolumeExpansionSuite5/PVCUnattachment.png[0m
			BoxPlot:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/VolumeExpansionSuite5/PVCUnattachment_boxplot.png[0m
		    PodCreation:
			Avg: 25.807500136s
			Min: 25.807500136s
			Max: 25.807500136s
			Histogram:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/VolumeExpansionSuite5/PodCreation.png[0m
			BoxPlot:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/VolumeExpansionSuite5/PodCreation_boxplot.png[0m
		    PodDeletion:
			Avg: 4.492857325s
			Min: 4.492857325s
			Max: 4.492857325s
			Histogram:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/VolumeExpansionSuite5/PodDeletion.png[0m
			BoxPlot:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/VolumeExpansionSuite5/PodDeletion_boxplot.png[0m
			EntityNumberOverTime:
	[36m/root/certify-csm/reports/reports/test-run-fe74a2e8/VolumeExpansionSuite5/EntityNumberOverTime.png[0m
```

### Additional Details

This configuration is configured to use the CSM Authorization version 1.6.0.

**Comment by anandrajak1** [comment]
@marbleman2 - Can you please let us know the CSI driver version used for this qualification V2.6.1 ?
 we are at v2.12.0 version.

**Comment by marbleman2** [comment]
Lateef,

Can you please provide the CSI Driver that was used for the self-cert sir?

Jason Marble
Senior Principal Engineer, TBC Account Managed Services
Dell Technologies | Services
mobile + 1 336 455 3892
***@***.******@***.***>
***@***.******@***.***>

NEW: http://sba.web.boeing.com

From: AnandatDell ***@***.***>
Sent: Friday, November 8, 2024 10:42 AM
To: dell/csm ***@***.***>
Cc: EXT-Marble, Jason M ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [dell/csm] [Qualification]: PowerScale CSI v2.6.1 on OpenShift 4.14 (Issue #1513)


EXT email: be mindful of links/attachments.





@marbleman2<https://github.com/marbleman2> - Can you please let us know the CSI driver version used for this qualification V2.6.1 ?
we are at v2.12.0 version.

**Comment by marbleman2** [comment]
The helm repo used was from:

https://github.com/dell/csi-powerscale.git

precisely ==> https://github.com/dell/csi-powerscale/tree/v2.6.1

This is the version that we used.

Thanks,

Lateef O.

From: EXT-Marble, Jason M ***@***.***>
Sent: Friday, November 8, 2024 10:31 AM
To: dell/csm ***@***.***>; dell/csm ***@***.***>; EXT-Onifade, Lateef ***@***.***>
Cc: Mention ***@***.***>
Subject: RE: [EXTERNAL] Re: [dell/csm] [Qualification]: PowerScale CSI v2.6.1 on OpenShift 4.14 (Issue #1513)

Lateef,

Can you please provide the CSI Driver that was used for the self-cert sir?

Jason Marble
Senior Principal Engineer, TBC Account Managed Services
Dell Technologies | Services
mobile + 1 336 455 3892
***@***.******@***.***>
***@***.******@***.***>

NEW: http://sba.web.boeing.com

From: AnandatDell ***@***.******@***.***>>
Sent: Friday, November 8, 2024 10:42 AM
To: dell/csm ***@***.******@***.***>>
Cc: EXT-Marble, Jason M ***@***.******@***.***>>; Mention ***@***.******@***.***>>
Subject: [EXTERNAL] Re: [dell/csm] [Qualification]: PowerScale CSI v2.6.1 on OpenShift 4.14 (Issue #1513)


EXT email: be mindful of links/attachments.





@marbleman2<https://github.com/marbleman2> - Can you please let us know the CSI driver version used for this qualification V2.6.1 ?
we are at v2.12.0 version.

**Comment by anandrajak1** [comment]
Ok thanks

**Comment by marbleman2** [comment]
Please let us know if the qualification was successful as I don’t see a mention.

Jason Marble
Senior Principal Engineer, TBC Account Managed Services
Dell Technologies | Services
mobile + 1 336 455 3892
***@***.******@***.***>
***@***.******@***.***>

NEW: http://sba.web.boeing.com

From: AnandatDell ***@***.***>
Sent: Tuesday, November 12, 2024 6:12 PM
To: dell/csm ***@***.***>
Cc: EXT-Marble, Jason M ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [dell/csm] [Qualification]: PowerScale CSI v2.6.1 on OpenShift 4.14 (Issue #1513)


EXT email: be mindful of links/attachments.





Closed #1513<https://github.com/dell/csm/issues/1513> as completed.
